packages/@uppy/file-input/src/FileInput.tsx

```diff
@@ -59,14 +76,15 @@ export default  class FileInput extends UIPlugin {
     // ___Why not use value="" on <input/> instead?
     //    Because if we use that method of clearing the input,
     //    Chrome will not trigger change if we drop the same file twice (Issue #768).
+    // @ts-expect-error yes
     event.target.value = null // eslint-disable-line no-param-reassign
   }
 
-  handleClick () {
-    this.input.click()
+  private handleClick() {
+    this.input!.click()
   }
 
-  render () {
+  render(): ComponentChild {
```

Comment:
On other PRs I'm using the following:
```suggestion
  render(): JSX.Element {
```
Not sure if there's a difference between the two, but we should probably pick one or the other to be consistant

Reply:
discussed this with @Murderlon in another PR, but tldr: JSX.Element is only that (a tag), ComponentChild can be anything that can be rendered (string, null, jsx.element, array of elements etc), so it's better to type render with ComponentChild. i believe it's the same as React.ReactNode in react. see https://www.totaltypescript.com/jsx-element-vs-react-reactnode

Reply:
Makes sense, I'll update the code that uses `JSX.Element`